frontend/src/common/ReviewWithComment.tsx

```diff
@@ -113,7 +120,6 @@ export default function ReviewWithComment({
                   size='small'
                   InputProps={{
```

Comment:
`InputProps` doesn't need to be specified at all as it's already covered by `{...params}` above